packages/fetchai/connections/soef/connection.py

```diff
@@ -77,7 +126,14 @@ def __init__(
         :param soef_port: the SOEF port.
         :param excluded_protocols: the protocol ids excluded
         :param restricted_to_protocols: the protocol ids restricted to
+        :param chain_identifier: supported chain id
         """
+        if (
+            chain_identifier is not None
+            and chain_identifier not in self.SUPPORTED_CHAIN_IDENTIFIERS
+        ):
+            raise ValueError("Unsupported chain_identifier")
```

Comment:
Print the list of supported identifiers in the error message please

Reply:
done